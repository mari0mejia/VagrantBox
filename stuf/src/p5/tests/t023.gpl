int i=99;
on akey 
{
  for(i=0; i<5; i++)
    print("Counting: " + i);
  print("Confirm For::execute() ends with:");
  print("    if(next!=nullptr) next->execute();");
}
