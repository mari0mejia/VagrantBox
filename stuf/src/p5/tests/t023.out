gpl.cpp::main()
  input file(tests/t023.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int i = 99
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[5]: Counting: 0
print[5]: Counting: 1
print[5]: Counting: 2
print[5]: Counting: 3
print[5]: Counting: 4
print[6]: Confirm For::execute() ends with:
print[7]:     if(next!=nullptr) next->execute();

User quit program. Printing the symbol table.
int i = 5
